### ISSUE TITLE
More deprecations from trio 0.15.1

### ISSUE THREAD

**Issue by space88man** [agronholm/anyio#109]
I noted this for trio 0.15.1 
```
anyio            1.3.0
asks             2.3.7
trio             0.15.1
anyio/_backends/_trio.py:5: TrioDeprecationWarning: trio.hazmat is deprecated since Trio 0.15.0; use trio.lowlevel instead (https://github.com/python-trio/trio/issues/476)

anyio/_backends/_trio.py:243: TrioDeprecationWarning: subclassing trio.Condition is deprecated since Trio 0.15.0; use composition or delegation instead (https://github.com/python-trio/trio/issues/1044)
```

**Comment by agronholm** [comment]
Those were fixed by PR #106, weren't they?

**Comment by space88man** [comment]
Ah! I was using the one in PyPI and forgot to check the current master.

**Comment by agronholm** [comment]
I've released AnyIO 1.3.1 now which includes this fix.
